kepler/tasks.py

```diff
@@ -20,6 +21,7 @@
 from ogre.xml import FGDCParser
 from lxml import etree
 import pysolr
+import requests
```

Comment:
Let's use a separate PR for organizing imports, just to keep things clean and easy to follow.